src/lib/npmCommand.ts

```diff
@@ -78,28 +79,86 @@ export class NpmCommand {
     return this.npmPkgPath;
   }
 
-  private static npmCli(): string {
+  /**
+   * Return a executable path to this modules reference to npm as
+   * <path to node executable> <path to npm-cli.js>
+   *
+   * @private
+   */
+  private static npmCli(root: string = undefined): string {
+    const nodeBinPath = NpmCommand.findNodeBin(root);
     const pkgPath = NpmCommand.npmPackagePath();
     const pkgJson = fs.readJsonSync(pkgPath) as NpmPackage;
     const prjPath = pkgPath.substring(0, pkgPath.lastIndexOf(path.sep));
-    return path.join(prjPath, pkgJson.bin['npm']);
+    return `"${nodeBinPath}" "${path.join(prjPath, pkgJson.bin['npm'])}"`;
+  }
+
+  /**
+   * Locate node executable and return its full path.
+   * First see if node is on the path, if so, use unqualified name.
+   * If not on the path, try to locate the node version installed with sfdx.
+   * If found, return full path to the executable
+   * If sfdx or node executable cannot be found, an exception is thrown
+   *
+   * @private
+   */
+  private static findNodeBin(root: string = undefined): string {
+    let sfdxPath;
+    if (root) {
+      sfdxPath = root;
+    } else {
+      throw new SfdxError('Plugin root dir is not set', 'PluginRootNotSet');
+    }
+    // find node within sfdx installation
+    const sfdxBinDirPaths = NpmCommand.getSfdxBinDirs(sfdxPath);
+    if (sfdxBinDirPaths && sfdxBinDirPaths.length > 0) {
+      const nodeBinPath = shelljs
+        .find(sfdxBinDirPaths)
+        .filter((file) => {
+          const fileName = path.basename(file);
+          const stat = fs.statSync(file);
+          const isExecutable = !stat.isDirectory();
+          return isExecutable && (process.platform === 'win32' ? fileName === 'node.exe' : fileName === 'node');
+        })
+        .find((file) => file);
+      if (nodeBinPath) {
+        return fs.realpathSync(nodeBinPath);
+      }
+    }
+    // check to see if node is installed
```

Comment:
if node is installed and on the path, what's the logic for not using it before trying to find sfdx's built-in node?

Reply:
We know `sfdx`'s built-in node will work with our plugins, the installed version of node could be a bad or old version, or even a newer version that we don't support yet (like when we had that issue with REST deploys and node 16).

Reply:
makes sense